### PR TITLE
Bedrock integration

### PR DESCRIPTION
**Pull request by gaya3-zipstack**
## What

Integrate Amazon bedrock as a LLM adapter

## Why

To support LLM models offered by Amazon Bedrock

## How

By adding Bedrock as an adapter

## Relevant Docs

-

## Related Issues or PRs

-

## Dependencies Versions / Env Variables

-

## Notes on Testing

Tested with model mazon.titan-text-express-v1

## Screenshots

image

image

image

## Checklist

I have read and understood the [Contribution Guidelines]().

### CODE REVIEW

**Review by tahierhussain — APPROVED** [review]

File: src/unstract/sdk/adapters/llm/bedrock/src/bedrock.py [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+                aws_access_key_id=self.config.get(Constants.ACCESS_KEY_ID),
+                aws_secret_access_key=self.config.get(Constants.SECRET_ACCESS_KEY),
+                region_name=self.config.get(Constants.REGION_NAME),
+                timeout=float(
+                    self.config.get(Constants.TIMEOUT, LLMKeys.DEFAULT_TIMEOUT)
+                ),
+                max_retries=int(
+                    self.config.get(Constants.MAX_RETIRES, LLMKeys.DEFAULT_MAX_RETRIES)
+                ),
+                temperature=0,
```
Comment (muhammad-ali-e) [inline src/unstract/sdk/adapters/llm/bedrock/src/bedrock.py:74]: Will this allways zero?

**Review by muhammad-ali-e — APPROVED** [review]
LGTM- added minor commend.
- Note: It would be better if we add something (Ex: anything related to the configurauions) in Readme.md

File: src/unstract/sdk/adapters/llm/bedrock/src/bedrock.py (hunk 1 above)
Comment (gaya3-zipstack) [reply src/unstract/sdk/adapters/llm/bedrock/src/bedrock.py:74]: Yes. For all adapters, @arun-venkataswamy has recommended this to be kept zero to not allow LLMs to get creative. Thereby consistency will be kept intact in the responses @muhammad-ali-e

**Review by kirtimanmishrazipstack — APPROVED** [review]
LGTM

**Pull request merged**